### PR TITLE
Added batched APIs in ProducerStateTable.

### PR DESCRIPTION
**Pull request by mint570**
swss-common: Added batched APIs in ProducerStateTable.
HLD: https://github.com/Azure/SONiC/pull/959

Signed-off-by: Runming Wu [runmingwu@gmail.com](mailto:runmingwu@gmail.com)

### CODE REVIEW

**Comment by mint570** [comment]
@qiluo-msft 
@prsunny

File: common/producerstatetable.h [hunk 1]
```diff
@@ -36,6 +36,17 @@ class ProducerStateTable : public TableBase, public TableName_KeySet
     %}
 #endif
 
+    // Batched version of set() and del(), for Google usage only.
+    // The batched methods are not virtual. They don't include (or use) op and
+    // prefix. They are written for specific use case only.
```
Comment (qiluo-msft) [inline common/producerstatetable.h:41]: If you want to add new feature in this repo, it must be implemented for all use cases. This one need cover `op` and `prefix` if technically possible.

Comment (mint570) [reply common/producerstatetable.h]: I deleted the Google specific comments. PTAL.

**Review by qiluo-msft — APPROVED** [review]

**Comment by qiluo-msft** [comment]
Could you merge latest master? The build issue should be fixed.

**Comment by mint570** [comment]
> Could you merge latest master? The build issue should be fixed.

Need to make some change for https://github.com/Azure/sonic-swss-common/pull/568.
Should be good now.
Please re-approve after the test finishes.
Thanks.

**Review by qiluo-msft — APPROVED** [review]

**Comment by mint570** [comment]
@qiluo-msft 
Thanks for your approval. The vs test didn't run after you approved.
This pr should be good to merge.

**Comment by mint570** [comment]
@qiluo-msft 
Hi Qi, can we merge this pr now if it looks good? Thanks.

**Pull request merged**